core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -704,7 +706,8 @@ public PipeLineResult processMessageWithExceptions(String messageId, Message mes
 
 	// technically, a Receiver is not mandatory, but no useful adapter can do without it.
 	/**
-	 * Register a receiver for this Adapter
+	 * Receives incoming messages. If an adapter can receive messages from multiple sources,
+	 * then add a receiver for each source.
```

Comment:
Je moet alleen voor elke source een receiver toevoegen als dat nodig is. Twee httpclients kunnen best dezelfde receiver gebruiken

Reply:
De term "source" was het probleem. Ik spreek nu van "channel".